src/view/Dialog.less

```diff
@@ -0,0 +1,56 @@
+@import (reference) '~nav-frontend-core/less/core';
+
+.dialog {
+  background: @navLysGra;
+  display: flex;
+  flex-direction: column;
+}
+
+.dialog-top {
+  flex-grow: 1;
+  display: flex;
+  flex-direction: column;
+  justify-content: space-evenly;
+  align-content: space-around;
+  background: @navLysGra;
+  margin: 10px;
+
+}
+
+.dialog-bottom {
+  flex-grow: 1;
+  background: @navLysGra;
+ }
+
+.aktivitet {
+  flex-grow: 1;
+}
+.aktivitet-chevron  {
+  font-weight: bold;
+}
+
+.venter-pa {
+  flex-grow: 1;
```

Comment:
`flex-grow` og `align-content` tror jeg ikke har noen effekt her. Så de ville jeg vurdere å fjerne.

Videre bruker vi vanligvis å prøve å unngå `margin-top`, og legger legge `margin-bottom` på elementet ovenfor. 
Da kan man bruke utility-klassene fra nav-frontend `blokk-m`, `blokk-s` etc for å avstander som er spesifisert i designsystemet

Reply:
Gjort ( flex-grow og align-content:  borte, ) men margin ble verre pga de 2 elementene over som ikke lenger har en felles klasse. Jeg foreslår å ta tilbake `aktivitet` og legge `margin-bottom` på den.